crates/symbolicator/src/sources.rs

```diff
@@ -137,17 +137,36 @@ where
         .map_err(|e| D::Error::custom(format!("region: {}", e)))
 }
 
+#[derive(Clone, Copy, Debug, Deserialize, Serialize)]
+#[serde(rename_all = "lowercase")]
+pub enum AwsCredentialsProvider {
```

Comment:
Do you mind giving this a doc comment as well?  I'd even include a link to https://docs.aws.amazon.com/AmazonECS/latest/developerguide/task-iam-roles.html or some other relevant AWS docs as you did in the PR description.

Reply:
Done.